res/database/Default/user/therob.xml

```diff
@@ -25,65 +27,77 @@
 					<title>
 						<de>Wirschi und der Tante Emma Laden</de>
 						<en>Gruffy and the corner store</en>
+						<pl>Gruffy i sklep na rogu</pl>
 					</title>
 					<description>
 						<de>Kommissar Wirsch ermittelt bei einem Kaugummiklau. Wer kaut denn da auf einmal im Dorf?</de>
 						<en>Inspector Gruff investigates a chewing gum theft. Who is stealing and chewing gum in the village all of a sudden?</en>
+						<pl>Inspektor Gruff bada sprawę kradzieży gumy do żucia. Kto nagle kradnie i żuje gumę w wiosce?</pl>
```

Comment:
German version is a pun where stealing and chewing are (almost) the same word, maybe you have something like that in Polish.

Reply:
A tough case

Reply:
adapted